core/src/main/java/nl/nn/adapterframework/jdbc/migration/Migrator.java

```diff
@@ -38,7 +39,7 @@
  * @since	7.0-B4
  *
  */
-public class Migrator extends JdbcFacade {
+public class Migrator extends JdbcFacade implements AutoCloseable {
```

Comment:
The Spring term would be `DisposableBean` instead of AutoClosable.
Either way, since it's a Singleton bean neither will be called automatically.
I think it would be wise to add a comment somewhere explaining this behavior and the problem that occurs when the JDBC connection isn't closed.